kubectl-openebs/cli/command/get/volume_list.go

```diff
@@ -14,10 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package volume
+package get
```

Comment:
I've made a mess with packaging right now, I sort of copied how things were in kubectl and created a get package with a get.go, after this I'm a bit confused w.r.t should there be a single `volume_cmd.go` which will have all functionality talking to the commands & flags about `get`, `describe`, `delete`, `watch` (might get added later if useful).

Reply:
We can have a single file for now and as we move further, we can expand(move) the commands to different files. wdyt?

Reply:
I am planning to move around logic and files in upcoming PRs, will discuss sometime, I'm sort of blindly following what k8s has already done^(this may not be the best approach) as we are solving a much smaller problem right now and may expand to a larger use-case.

^See the structure here -> https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/kubectl/pkg/cmd the sig-cli probably went ahead with this structure because they've got a lot of sub-commands(like get, describe, explain, etc)